src/theory/fp/fp_word_blaster.h

```diff
@@ -106,7 +106,7 @@ class nodeWrapper : public Node
  * allowing them to be traced via GDB.
  */
 #ifdef CVC5_SYM_SYMBOLIC_EVAL
-  nodeWrapper(const Node& n) : Node(theory::Rewriter::rewrite(n)) {}
+  nodeWrapper(const Node& n) : Node(theory::rewrite(n)) {}
```

Comment:
I think this should be different...

Reply:
Hm, good catch, apparently this code is disabled?